src/tests/unittests/asn1.c

```diff
@@ -502,6 +620,11 @@ int main(void)
 			setup_sc_context, teardown_sc_context),
 		cmocka_unit_test_setup_teardown(torture_asn1_decode_entry_bit_string_ni,
 			setup_sc_context, teardown_sc_context),
```

Comment:
clang-format suggested the following change:


```
-			setup_sc_context, teardown_sc_context),
+						setup_sc_context, teardown_sc_context),
```

<!-- identifier: clang-format -->
<!-- File: src/tests/unittests/asn1.c -->